.scalafmt.conf

```diff
@@ -9,3 +9,4 @@ danglingParentheses.preset = true
 rewrite.rules = [AvoidInfix, SortImports, RedundantBraces, RedundantParens, SortModifiers]
 docstrings.style = Asterisk
 align.preset = none
+project.layout = StandardConvention
```

Comment:
This setting instructs scalafmt to apply the Scala 3 dialect to Scala 3 sources since this is the first time that this project has explicit Scala 3 sources.